tests/test_queryset.py

```diff
@@ -314,3 +314,7 @@ async def test_all_values(self):
     async def test_order_by_bad_value(self):
         with self.assertRaisesRegex(FieldError, "Unknown field badid for model IntFields"):
             await IntFields.all().order_by("badid").values_list()
+
+    async def test_get_raw_sql(self):
+        sql = IntFields.all().sql()
+        self.assertGreater(len(sql), 10)
```

Comment:
This testcase don't looks nice. It is unable to check that if a valid query has been created. We should compare that REAL concrete query was generated. Also need to check more query (e.g : with join, using function, etc ...)

Reply:
> Hi, thanks for this.
> 
> Could you please add yourself to `CONTRIBUTORS.rst` and a short description of the change in `CHANGELOG.rst`.
> 
> Also, could you please do a test for this?
> A simple `sql = Foo.all().sql()` would do.

I'm not sure about how to add test code for .sql(), so I ask for help, maybe just make a detail assert is enough?
```python
assert sql == 'SELECT .. FROM'
```

I think maybe we don't need to check more query, because we already test the function in other tests of queryset.

Reply:
As you said, I think `assertEqual(sql, 'SELECT ... FROM')` is better. Also I want to check that new method is strong in many cases. I think we need more opinion. @grigi What do you think about this one?

Reply:
We definitely want to test that it's a query.
A simple = would be ideal, with the caveat of different syntaxes.

Note that the SQL generated is going to be different between different dialects, so you may just test that it starts with `SELECT ` and has ` FROM ` etc...

Reply:
@grigi @lntuition I update the test, if we need more tests, I will do.

By the way, thank all your work for the awesome ORM.